Src/VimCore/CoreInterfaces.fs

```diff
@@ -3413,25 +3468,25 @@ type MotionBinding =
     /// Simple motion which comprises of a single KeyInput and a function which given 
     /// a start point and count will produce the motion.  None is returned in the 
     /// case the motion is not valid
-    | Simple of KeyInputSet * MotionFlags * Motion
+    | Static of KeyInputSet * MotionFlags * Motion
 
     /// Complex motion commands take more than one KeyInput to complete.  For example 
     /// the f,t,F and T commands all require at least one additional input.  The bool
     /// in the middle of the tuple indicates whether or not the motion can be 
     /// used as a cursor movement operation  

```

Comment:
These comments are slightly out of date, e.g. simple/complex vs. static/dynamic and referring to a obsolete bool.

Reply:
Thanks. Fixed.